source/API_Reference/Web_API_v3/bounces.apiblueprint

```diff
@@ -0,0 +1,88 @@
+---
+layout: page
+weight: 100
+title: Bounces API
+seo:
+  description: Manage your bounce list
+  title: Bounce API
+navigation:
+  show: true
+---
+FORMAT: 1A
+
+# Bounces
+
+
+## Bounces Collection [/v3/suppression/bounces?start_time={start_time}&end_time={end_time}]
+
++ Parameters
+
+    + start_time (optional, number, `1443651141`) ... Refers start of the time range when a bounce was created.
+    + end_time (optional, number, `1443651154`) ... Refers end of the time range when a bounce was created.
```

Comment:
So it's marked as timestamp in apiary but something in the docs compiler throws up when I write timestamp so now it's number.